youtube_dl/utils.py

```diff
@@ -183,7 +183,7 @@ def register_socks_protocols():
 ])
 
 PACKED_CODES_RE = r"}\('(.+)',(\d+),(\d+),'([^']+)'\.split\('\|'\)"
-JSON_LD_RE = r'(?is)<script[^>]+type=(["\'])application/ld\+json\1[^>]*>(?P<json_ld>.+?)</script>'
+JSON_LD_RE = r'(?is)<script[^>]+type=(["\'])?application/ld\+json\1?[^>]*>(?P<json_ld>.+?)</script>'
```

Comment:
Invalid. Either both quotes must be present or none. Not one of them.

Reply:
resolved for all three posibilities.